### ISSUE TITLE
[Bug]: Icon is missing from tour guide

### ISSUE THREAD

**Issue by SViN24** [bottlesdevs/Bottles#1627]
### Describe the bug

There is supposed to be a checkmark icon in the end of the tour guide and it is missing
![image](https://user-images.githubusercontent.com/51864332/173735350-3f3592d5-9c23-4b16-8fb6-bd3057f2ab8f.png)

### To Reproduce

remove ~/.var/app/com.usebottles.bottles
restart application and go through tour guide

### Package

Flatpak from Flathub

### Distribution

Fedora Kinoite Rawhide

### Health Check

```shell
Version: 2022.6.14-brescia
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: false
Graphics:
    vendors:
        intel:
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.19.0-0.rc1.20220610git874c8ca1e60b.18.fc37.x86_64
Distro:
    Name: GNOME
    Version: '"42 (Flatpak runtime)"'
Disk:
    Total: 4081889280
    Free: 4081741824
RAM:
    MemTotal: 7.6GiB
    MemAvailable: 4.7GiB
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.33'
Bottles_envs: null
```

### Display Protocol

X11

### Display Backend used by Bottles

X11

**Comment by TheEvilSkeleton** [comment]
FWIW I can't reproduce it on my end. Using Silverblue 36 on Wayland.

**Comment by TheEvilSkeleton** [comment]
I can reproduce it on Plasma on Arch Linux.

**Comment by jannuary** [comment]
`selection-mode-symbolic` is a part of a-i-t, so not our bug really.

**Comment by mirkobrombin** [comment]
Do you have a custom theme set up?

**Comment by SViN24** [comment]
> Do you have a custom theme set up?
> 
> ![image](https://user-images.githubusercontent.com/9479903/173754129-4d2a237d-fa82-457f-b47b-3ac39d540b5d.png)
> 
> 

This is running under KDE so it should be using breeze

**Comment by jannuary** [comment]
Right, yeah. We use icons from Adwaita, and if a particular theme is set to inherit Adwaita, it just works. 
![image](https://user-images.githubusercontent.com/27908024/173824461-2e5bdbcf-0170-4207-951c-4f0d8bd7135f.png)

Unfortunately, KDE in their limited wisdom decided to not do that - it only inherits from hicolor, which breaks about half the icons in the app.

**Comment by Matombo** [comment]
The workaround does not seem to be packaged with the flatpack release. Is this intentional?

**Comment by farline99** [comment]
I changed breeze color scheme file to inherit icons from Adwaita. But flatpak version of this app does not benefit from it. AUR version feels good and shows every icon. Can you fix this? Very frustrating issue :)

**Comment by digimezzo** [comment]
For those coming here a after a web search, the explanation above is not accurate. The evidence:

- When using the OS's default icon set, Bottles displays the icons. 
- When using a custom icon set, Bottles displays missing icons. 

Custom icon sets are usually installed in `~/.icons`, while default icons are located in `/usr/share/icons`. This data suggests that Bottles cannot read the icons when they are located in the `/home` directory. Which might be related to the sandboxed nature of Flatpak.

**The fix to this issue**: install the custom icons to `/usr/share/icons` instead of `~/.icons`.

**Comment by TheEvilSkeleton** [comment]
`~/.icons` is an unsupported and nonstandard path. You want to use `~/.local/share/icons` instead.

**Comment by digimezzo** [comment]
@TheEvilSkeleton My apologies. I wasn't aware of that. Thanks for pointing that out. When putting custom icons in `~/.local/share/icons`, the icons show up correctly in Bottles.
